package.json

```diff
@@ -27,6 +27,7 @@
   ],
   "main": "dist/index.js",
   "module": "src/index.js",
+  "types": "index.d.ts",
   "files": [
     "dist",
     "src/index.js"
```

Comment:
index.d.ts also needs to be added to files, or it won't be uploaded when publishing to npm